spec/requests/v6/assessments_controller_spec.rb

```diff
@@ -216,6 +219,7 @@ module V6
                   smod_allowance: 0,
                   main_home_equity_disregard: 0.0,
                   assessed_equity: 0.0,
+                  subject_matter_of_dispute: nil,
```

Comment:
don't think this should ever be nil - the schema doesn't allow for it. IMHO this defaults to false when not specified?

Reply:
Double checked and it seems to not suffer from the issues we had with the employed boolean